dev/com.ibm.ws.kernel.boot.core/src/io/openliberty/checkpoint/spi/CheckpointPhase.java

```diff
@@ -39,13 +39,22 @@ public enum CheckpointPhase {
      * Phase of startup after the applications have been discovered but no
      * code from the application has been executed.
      */
-    DEPLOYMENT;
+    DEPLOYMENT,
+    /**
+     * Phase that indicates there is no checkpoint being done.
+     */
+    INACTIVE(true);
```

Comment:
Update Javadoc for `CheckpointPhase` which currently says
> This enum is registered as a service _when a checkpoint operation is being done_.

Reply:
I've rebased to latest integration branch so you comment is not included in the change view.  But I updated the class doc to:

```
Phase which a checkpoint of the running process is being taken. 
This enum is registered as a service when running in an OSGi framework.
```